spec/models/geometry/edge_spec.rb

```diff
@@ -8,11 +8,19 @@
     expect(subject.length).to eq 14.142135623730951
   end
 
-  context 'with vector'
-  let(:vector) { [10, -10] }
+  describe '#offset_point_by_half_of_length' do
+    let(:vector) { [10, -10] }
 
-  it 'should offset the midpoint' do
-    expect(subject.offset_point_by_half_of_length(0)).to eq [5, 5]
-    expect(subject.offset_point_by_half_of_length(0.5)).not_to eq [5, 5]
+    context 'with default factor' do
+      it 'should offset the midpoint' do
+        expect(subject.offset_point_by_half_of_length(0.5)).to eq [7.5, 2.5]
+      end
+    end
+
+    context 'with a factor of  0,5' do
+      it 'should offset the midpoint' do
+        expect(subject.offset_point_by_half_of_length).to eq [5, 5]
+      end
+    end
```

Comment:
sadly the test description is now not matching the actual test .... where there is 0.5 the description says default and the other way around :(